lib/manager/terragrunt-version/extract.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { extractPackageFile } from './extract';
```

Comment:
```suggestion
import { getName } from '../../../test/util';
import { extractPackageFile } from './extract';
```

Reply:
My bad, missed the supplementary comment in the initial suggested change.

Was just testing this locally, but you beat me to it 😄